tray.js

```diff
@@ -57,7 +56,13 @@ module.exports.setUpTray = (app, win) => {
 				win.show();
 			},
 		},
-		...mainMenuTemplate(win, true, true),
```

Comment:
How about adding a tray option to show/hide the menu? This way, when the option "Hide menu" is enabled, it is possible to show again the menu without editing the config (using the tray menu).

Reply:
The problem is that it wont be synced with the main menu checkbox. 
and you can already show the menu when its hidden with `alt` key (or `escape` with in-app-menu)

  Are you sure I should add it?

Reply:
Good point, should be good as is then!